test/integration/features/cmd-oc-env.feature

```diff
@@ -0,0 +1,31 @@
+@cmd-oc-env @command
+Feature: Command oc-env
+Command oc-env helps the user to set path to oc binary so she can run the oc command within the VM host.
```

Comment:
This is not correct. The `oc` command does nor run with the Minishift VM. The command only helps to set the path to the oc binary so that user can use the oc cli against the OpenShift instance from command line.

Reply:
@LalatenduMohanty it talks about the usage of oc command withing the VM host, not the Minishift VM aka guest. Is it fine, or should I make it more clear somehow?